content/docs/tutorials/gcp/gcp-ts-slackbot.md

```diff
@@ -1,5 +1,7 @@
 ---
-title: "Slackbot for Posting Slack Mention Notifications
"
+title: "Slackbot for Posting Slack Mention Notifications
 | TypeScript"
```

Comment:
Unrelated to this change, but we should open an issue to strip the `^M` (carriage return)

Reply:
Issue here: https://github.com/pulumi/docs/issues/2312